### PR TITLE
[bitnami/kube-state-metrics] Add kube-state-metrics self monitor

### PR DESCRIPTION
**Pull request by Jack296s**
Signed-off-by: Jack Yang [qwedd296@gmail.com](mailto:qwedd296@gmail.com)
### Description of the change

Add kube-state-metrics self monitor

### Benefits

kube-state-metrics exposes its own general process metrics under `--telemetry-port` (default 8081)

### Applicable issues

- fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by dgomezleon — APPROVED** [review]
LGTM. Thanks for contributing

File: bitnami/kube-state-metrics/README.md [hunk 1]
```diff
@@ -197,8 +199,11 @@ The command removes all the Kubernetes components associated with the chart and
...
 | `serviceMonitor.labels`                         | Extra labels for the ServiceMonitor                                                                                                                                | `{}`                         |
+| `selfMonitor.enabled`                           | Creates a selfMonitor to monitor kube-state-metrics itself                                                                                                         | `false`                      |
+| `selfMonitor.telemetryPort`                     | Kube-state-metrics telemetry Port                                                                                                                                  | `8081`                       |
+| `selfMonitor.telemetryNodePort`                 | Kube-state-metrics Node Port                                                                                                                                       | `""`                         |
+
 
-Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example the following command sets the `replicas` of the kube-state-metrics Pods to `2`.
```
Comment (dgomezleon) [inline bitnami/kube-state-metrics/README.md:201]: There is a minor typo here. Could you check it?

**Review by dgomezleon — CHANGES_REQUESTED** [review]

File: bitnami/kube-state-metrics/README.md (hunk 1 above)
Comment (Jack296s) [reply bitnami/kube-state-metrics/README.md:201]: @dgomezleon 
I don't delete the sentence.
**bitnami-bot** delete the sentence automatically.

Comment (Jack296s) [reply bitnami/kube-state-metrics/README.md:201]: After I added the sentence back, bitnami-bot automatically deleted it again

Comment (dgomezleon) [reply bitnami/kube-state-metrics/README.md:201]: You are right. We are working on a fix [here](https://github.com/bitnami-labs/readme-generator-for-helm/pull/61)

**Comment by fmulero** [comment]
Sorry it was closed automatically by mistake.

File: bitnami/kube-state-metrics/README.md (hunk 1 above)
Comment (fmulero) [reply bitnami/kube-state-metrics/README.md:201]: Please add the line again. `readme-generator` shouldn't remove it again

**Comment by fmulero** [comment]
Closed automatically by mistake, sorry @Jack296s  again

**Review by dgomezleon — APPROVED** [review]

**Pull request merged**